go.mod

```diff
@@ -16,3 +16,5 @@ require (
 	k8s.io/kubectl v0.21.0
 	sigs.k8s.io/controller-runtime v0.9.0
 )
+
+replace github.com/openshift/api => github.com/sherine-k/api v0.0.0-20211112165433-12e91273cfc1
```

Comment:
Because the `replace` directive is still in here this PR should be marked as WIP.

Reply:
Sure :)